codegen/templates/templates.go

```diff
@@ -232,29 +232,52 @@ func ToCamel(s string) string {
 	if s == "_" {
 		return "_"
 	}
-	buffer := make([]rune, 0, len(s))
+	buf := bytes.NewBuffer(make([]byte, 0, len(s)))
```

Comment:
https://play.golang.org/p/QFS0VH1V90s

Reply:
Not needed, safe because all replacements are < 128

https://github.com/golang/lint/blob/06c8688daad7faa9da5a0c2f163a3d14aac986ca/lint.go#L734-L736